indicators/static_analysis_common_vulnerabilities.json

```diff
@@ -0,0 +1,23 @@
+{
+    "@context": "https://w3id.org/everse/rsqi#",
+    "@id": "https://w3id.org/everse/i/indicators/static_analysis_common_vulnerabilities",
+    "@type": "SoftwareQualityIndicator",
+    "name": "has static analysis for common vulnerabilities",
+    "description": "At least one static code analysis tool used includes rules or techniques to detect common vulnerabilities specific to the language or environment. Originally defined in https://www.bestpractices.dev/en/criteria/0.",
```

Comment:
Instead of "originally defined in..." you should have:
```
"source": {
        "@id": "https://www.bestpractices.dev/en/criteria/0",
        "url": "https://www.bestpractices.dev/en/criteria/0#0.static_analysis_common_vulnerabilities",
        "name": "FLOSS Best Practices Criteria (Passing Badge)"
    }
```